app/controllers/api/v1/gardens/env_measurements_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::Gardens::EnvMeasurementsController < ApplicationController
+	def index
+		garden = Garden.find_by(id: params[:id])
```

Comment:
@Mycobee Since it's displaying a single garden and not multiple gardens, should this endpoint be a `show` instead of an `index` ?

Then it could be `garden = Garden.find(params[:id])`

Reply:
The reason I went this route, when an ID is not found with find, it throws an error immediately.  Using find_by allows us to do more with a nil object for custom error handling.  I am currently working on taking this logic out of the controller to keep them skinny :)